Assets/Scripts/Utilities/Utils.cs

```diff
@@ -87,7 +98,13 @@ public static Vector3 CalculateHorizonPosition(float RA, float radianDec, float
 
         if (float.IsNaN(Alt)) Alt = 0;
         if (float.IsNaN(Azm)) Azm = 0;
-        return CalculatePositionByAzAlt(Azm, Alt, radius);
+        return new AltAz {Altitude = Mathf.Rad2Deg * Alt, Azimuth = Mathf.Rad2Deg * Azm};
+    }
+    [Obsolete]
```

Comment:
Just curious what this annotation does, and why it's useful to keep the older function signature around.

Reply:
Oh, Never mind, I think I see.  :)